rpcd/playbooks/verify-maas.yml

```diff
@@ -23,7 +23,7 @@
         . {{ maas_venv_bin }}/activate
         {% endif %}
         {{ maas_rpc_scripts_dir }}/rpc-maas-tool.py verify-created \
-        --entity {{ inventory_hostname }} \
+        --entity {{ inventory_hostname }}{{ maas_fqdn_extension }} \
```

Comment:
Another important thing to note. The template files that are linked in the bug report have nothing to do with the entity name - so I believe that is adding confusion. Those are the label names for the alarms/checks - these could be anything, e.g. "mymonitor" and the entity name would still remain the same.

The crux of the issue is that the entity is not being found correctly, Harry's final 2 paragraphs sum up what needs to be changed, as it is only the verify that is failing. If the entity was not found at creation time the maas play would fail earlier and monitors would not be created.